test/ClauseResultsBuilder.test.ts

```diff
@@ -428,5 +462,269 @@ describe('ResultsHelper', () => {
       const relevanceMap = ClauseResultsBuilder.buildPopulationRelevanceMap(populationResults);
       expect(relevanceMap).toEqual(expectedRelevanceMap);
     });
+
+    describe('multiple ipps', () => {
+      const group: fhir4.MeasureGroup = {
+        population: [
+          {
+            id: 'ipp-1',
+            code: {
+              coding: [
+                {
+                  system: 'http://terminology.hl7.org/CodeSystem/measure-population',
+                  code: 'initial-population'
+                }
+              ]
+            },
+            criteria: {
+              language: 'text/cql',
+              expression: 'ipp'
+            }
+          },
+          {
+            id: 'ipp-2',
+            code: {
+              coding: [
+                {
+                  system: 'http://terminology.hl7.org/CodeSystem/measure-population',
+                  code: 'initial-population'
+                }
+              ]
+            },
+            criteria: {
+              language: 'text/cql',
+              expression: 'ipp2'
```

Comment:
Here too for numerator

Reply:
The valueString in that extension refers to the `id` of the referenced population, not the criteria expression.